inspirehep/utils/references.py

```diff
@@ -95,6 +95,9 @@ def map_refextract_to_schema(extracted_references, source=None):
     result = []
 
     for reference in extracted_references:
+        # Skip references without author
+        if not reference.get('author'):
+            continue
```

Comment:
I'm sorry if I missed the conversation, but is it really what we want to achieve? Aren't we loosing references?

Reply:
no, this is absolutely not what we want. We don't care about authors in references. If there are not authors in references, we should just skip adding authors (can probably be fixed in `ReferenceBuilder.add_refextract_authors_str`).